src/world/area_kmr/kmr_22/kmr_22_2_npc.c

```diff
@@ -0,0 +1,809 @@
+#include "kmr_22.h"
+
+#include "world/common/npc/StarSpirit.h"
+#include "world/common/npc/KoopaBros.h"
+#include "world/common/enemy/complete/ShyGuy.h"
+
+MAP_RODATA_PAD(1,npc)
+
+NpcSettings N(NpcSettings_Dummy) = {
+    .height = 24,
+    .radius = 24,
+    .level = 99,
+};
+
+// Kolorado?
+NpcSettings N(NpcSettings_Unused) = {
+    .height = 40,
+    .radius = 24,
+    .level = 99,
+};
+
+#include "npc/lava_piranha.inc.c"
+
+#include "npc/chapter_1.inc.c"
+#include "npc/chapter_2.inc.c"
+#include "npc/chapter_3.inc.c"
+#include "npc/chapter_4.inc.c"
+#include "npc/chapter_5.inc.c"
+#include "npc/chapter_6.inc.c"
+#include "npc/chapter_7.inc.c"
+#include "npc/chapter_8.inc.c"
+
+StaticNpc N(NpcData_Chapter1)[] = {
```

Comment:
It's a shame that the npc lists come after the block of includes

Reply:
Its pretty standard in the map data for the NpcData structs to all come together after the npc scripts. Additionally, the NpcSettings structs are usually found first (before the scripts) as part of some set of includes.